src/yaml-tools.js

```diff
@@ -11,8 +13,20 @@ const LOG_LEVEL = {
   validate: level => ['information', 'warning', 'failure'].includes(level)
 };
 
-yamlLibrary = {
-  globalDoc: {},
+const cleanDocsData = docs => {
+  let lines = docs.split('\n');
+  let cleaned = '';
+  lines.forEach(line => {
+    let parts = line.split('#');
+    cleaned += `${parts[0]}\n`;
+  });
+  return cleaned;
+};
```

Comment:
Is this just to remove a comment at the end of the line?

Reply:
Yes.  This allows for comment removal and prevent a line like `#---` from slipping in.